### DISCUSSION TITLE
Can I close the Chia GUI wihtout canceling my plotting.

### DISCUSSION THREAD

**Discussion by skeg43** [ericaltendorf/plotman#536] [Q&A]
Hello, I am currently happily plotting with plotman (thanks), with my Chia GUI open so I can see farming status. I have tried to go to multiple github sites on my browser and I can't. I saw somewhere that the CHIA Gui blocks it. I want to try and close the Chia GUI so I can go get the Verus coin miner from github. 

Can I close the Chia GUI without stopping the plotting that plotman is doing?

**Comment by cyperbg** [comment]
Yes, you can. The plotting will continue.

**Reply by leonidcan** [reply]
Windows Chia GUI will stop all chia services (it does warn about this)
